examples/marathon.l5d

```diff
@@ -0,0 +1,17 @@
+namers:
+- kind:      io.l5d.experimental.marathon
+  prefix:    /io.l5d.marathon
+  host:      marathon.mesos
+  port:      80
+  uriPrefix: /marathon
```

Comment:
I don't understand why the api is prefixed with /marathon, or what I would do to change it away from the default of /marathon. Adding docs to config.md might be good to do as part of this change